vendor/github.com/openshift/cluster-api-provider-libvirt/pkg/apis/libvirtproviderconfig/v1beta1/libvirtmachineproviderconfig_types.go

```diff
@@ -33,6 +33,7 @@ type LibvirtMachineProviderConfig struct {
 	NetworkUUID              string     `json:"networkUUID"`
 	Autostart                bool       `json:"autostart"`
 	URI                      string     `json:"uri"`
+	PoolPath                 string     `json:"poolPath"`
```

Comment:
You cannot add this field. This is vendored code. The machine-api-operator is not responsible for creating libvirt pools. As such, this is not be an appropriate field to add to the machine spec, anyway.